### PR TITLE
[AWS S3] MC-38901: Add release notes

### PR DESCRIPTION
**Pull request by shiftedreality**
## Purpose of this pull request

This pull request (PR) adds an entry for the AWS S3 Remote Storage feature to the Open Source release notes.

It already exists in the Commerce release notes.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.4/release-notes/open-source-2-4-2.html

### CODE REVIEW

File: src/guides/v2.4/release-notes/open-source-2-4-2.md [hunk 1]
```diff
@@ -142,6 +142,12 @@ See the following articles for updates on features and changes for this release:
 
 *  [Yotpo Product Reviews]({{ site.baseurl }}/extensions/vendor/yotpo/release-notes.html)
 
+### AWS S3 support for the file storage
```
Comment (jeff-matthews) [inline src/guides/v2.4/release-notes/open-source-2-4-2.md:145]: Shouldn't this match the entry in the Commerce release notes?

```
### AWS S3 support enhancements

Amazon Simple Storage Service (AWS S3) support has been enhanced to include support for:

*  Object storage and future extensibility

*  [Storing media files]({{ page.baseurl }}/config-guide/remote-storage/config-remote-storage.html) on AWS S3

Support for AWS S3 has been added to all modules including B2B, PageBuilder, and Adobe Stock Integration.
```

Comment (shiftedreality) [reply src/guides/v2.4/release-notes/open-source-2-4-2.md:145]: Interesting, I have not seen that.

Don't we need to mention it in OpenSource if that's available there?

Comment (jeff-matthews) [reply src/guides/v2.4/release-notes/open-source-2-4-2.md:145]: Yes, if it's available in both Open Source and Commerce, we need duplicate entries in both files.

Comment (shiftedreality) [reply src/guides/v2.4/release-notes/open-source-2-4-2.md:145]: Thanks, then I will close my PR

**Comment by jeff-matthews** [comment]
Waiting for confirmation from Product Management.

**Review by jeff-matthews — APPROVED** [review]
Confirmed by Engineering Manager.

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**